### DISCUSSION TITLE
Collapsing problem

### DISCUSSION THREAD

**Discussion by insulae** [rappasoft/laravel-livewire-tables#708] [Q&A] [answered]
I'm new using laravel-livewire-tables and with laravel too. I'm trying to get two columns to show "collapsed" the columns start collapsed I can see and i can change the "collapse" button but It doesn't do anything, no error either,  I need to put something else to be able to see the data of the collapsed columns?
the code:
```

class CategoriaTabla extends DataTableComponent
{
    public $model = Categoria::class;

    public function configure(): void
    {
        $this->setPrimaryKey('id');
        $this->setSearchDebounce(1000);
    }

    public function columns(): array
    {
        return [
            column::make('ID', 'id')->sortable(),
            Column::make('nombre')
                ->sortable()
                ->searchable()
                ->format(function($value, $row, Column $column) {
                    return '<strong>'.$row->nombre.'</strong>';
                })->html(),
            Column::make('descripcion')
                ->sortable()
                ->collapseOnTablet()
                ->searchable(),
            Column::make('Usuario', 'user.name')
                ->collapseOnTablet(),
            LinkColumn::make('Acciones')
                ->title(fn($row) => 'Edit')
                ->location(fn($row) => route('categoria.edit', $row))
        ];
    }
```

Thanks for the help and for this amazing piece of code :-)

**Comment by rappasoft** [comment, marked as answer]
My gut feeling is you didn't tell tailwind to look at the table files to include them and not purge their classes.

**Reply by insulae** [reply]
If I go via PC widescreen I see the column and the data, but in table mode only appear the "collapse button" and when i do click  in the button the button change but the data not appear .
If I remove the option ->collapseOnTablet() and don't touch anything more the column appear in Tablet.

I made a php artisan optimize and still have the same result.

**Reply by rappasoft** [reply]
Check your console, that functionality works on Alpine.

**Reply by insulae** [reply]
![image](https://user-images.githubusercontent.com/18574307/162766900-6873fff1-4e89-4d57-b6ca-4f0eb37b13e5.png)

You have right, I have this error:
`Uncaught SyntaxError: Identifier 'started' has already been declared`

I put the table on a clean layout (I use the jetstream layout <x-app-layout>) and now its works.

the problem its in the app.blade.php I have at the end:
```

        @livewireScripts
    </body>
</html>
```
If I remove @livewireScripts the table works but the other livewire components not. Is there a way for them to coexist?

**Reply by rappasoft** [reply]
Look at the source of the demo to see how they are loaded.

**Reply by insulae** [reply]
Ok, its working, the error was I loaded the layout via tag and also passed the layout to the view with ->layout(). :-@ 
I would have spent 100 years looking for the problem if you didn't mark me the Alpine error.

Thanks Anthony for your time and for your knowledge.

**Reply by edikurniawan-dev** [reply]
@rappasoft can you make collapsing on desktop mode, please
